build.gradle

```diff
@@ -208,6 +209,55 @@ ext {
 
 ext['snakeyaml.version'] = '2.0'
 
+//here we are going to store swagger files
+def swaggerList = []
```

Comment:
I think previous generator was using useSpringboot3 for some validation in the courthouse api. if we are removing It we need to go back and add validaiton.

Reply:
we aren't generating any APIs here, just POJOs